src/main/io/serial.h

```diff
@@ -78,6 +78,7 @@ extern const uint32_t baudRates[];
 
 // serial port identifiers are now fixed, these values are used by MSP commands.
 typedef enum {
+    SERIAL_PORT_ALL = -128,
```

Comment:
Probably makes more sense to use -2 for this, so that parameters that should allow 'all' will have a continuos range.

Reply:
I don't expect that "ALL" will ever be useful as a setting as it's only used as an artificial "broadcast" identifier, but no problem. I'll push an update shortly.